oat-sa/tao-core-ui-fe
# Feature/nex 149 add babel setup

Ensure the sources can be written according the JavaScript coding guide in ES2015 and the generated AMD modules are compatible in `dist`.
This PR setup Babel to transpile the modules

How to test : 
```
npm i
npm run build
npm test
```
The file `src/component.js` has received minor changes to prove the transpilation works, so you can verify it in `dist`
